src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java

```diff
@@ -48,6 +48,10 @@
  *   --output tumor.seg
  * </pre>
  *
+ * While the tangent normalized (tn) data are log2 transformed, as indicated by the --log2Input true option, 
```

Comment:
I avoid use of "input" "output" in documentation. Also, this description refers specifically to what is happening in the two example commands immediately above. So, how about
```
 * In these example commands, while the tangent normalized (tn) coverage data are log2 transformed, as indicated by the --log2Input true option, 
 * the resulting segment means in the .seg file are not. PerformSegmentation accepts either non-log2 (specify --log2Input false) and log2 data.
 * However, {@link NormalizeSomaticReadCounts}, which calculates pretangent and tangent normalized coverage, can only produce log2-transformed counts.
```